run-tests-editmode-osx-editor.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -eo pipefail
+
+EDITOR_VERSION="2017.4.33f1"
+BOKKEN_UNITY="/Users/bokken/${EDITOR_VERSION}/Unity.app/Contents/MacOS/Unity"
+HUB_UNITY="/Applications/Unity/Hub/Editor/${EDITOR_VERSION}/Unity.app/Contents/MacOS/Unity"
+
+if [[ -f ${BOKKEN_UNITY} ]]; then 
+    UNITY=${BOKKEN_UNITY}
+else
+    UNITY=${HUB_UNITY}
+fi 
+
+pushd $(dirname "${0}") > /dev/null
+BASETPATH=$(pwd -L)
+popd > /dev/null
+
```

Comment:
What is the purpose of the pushd and popd here?

Reply:
not sure. was copy pasta :).  I can remove it.  Seems redundant.

Reply:
I think it's a safeguard against `pwd` directory not being `.` and also guarding against relative paths.  But may not be needed.

Reply:
Then let's keep it. Looks good to me.